_posts/2016-11-17-herding-hackers.md

```diff
@@ -0,0 +1,42 @@
+
+## Tip 5 - Start small, start slick
+Its easy to get tempted into starting a large public bounty programme, but I would strongly advise you to start small and scale slow - try testing the waters with an internal/private bug bounty programme, or a fixed PoC/pilot with a set bounty pot and a small number of hand-picked hackers. It's also vital to have a dedicated budget agreed and put aside, with the ability to quickly clear funds. Since no pot of money is infinite, it's important also to limit your exposure somehow and maintain control over likely expenditure. It may be a good idea, at least initially, to run for a defined trial period, or structure your programme around bursts, sprints or "seasons" so that you have an on-season and an off-season for bug hunting, giving you time in between to clear down vulnerability backlogs and get ready for the next onslaught.
+
+## Tip 6 - Keep your hackers happy
+Your hackers are an amazing resource. And they're all people, each of whom have slightly different motivations, methods of engagement, bug reporting style and expectations. If you run a bug bounty programme, you're going to swiftly recognise that its not about bug management, its about people management. The launch of a bug bounty is an amazing way of engaging with other teams throughout your organisation on a positive basis and building a security-focused community. No longer are you a cost centre or the team that likes to say "no", suddenly you're the source of bountiful rewards! This can be a real boon that you can build upon.
+
+## Tip 7 - One touch
+Templates. Just templates. You're going to need to manage large volumes of tickets and find yourself making the same statements over and over. Save your sanity and as soon as the trends become clear write a few, high quality boilerplate responses for different scenarios. Run them past your teammates, hone them, and use them.
+
+## Tip 8 - Management lives matter too
+Its easy to get swept up in the enthusiasm of operating a bug bounty programme but there may be a long list of people that you need to get support from for buy-in both at launch and on an ongoing basis. Are senior management happy at the increased risk profile of operating a bug bounty programme? What about your external comms team? Legal teams? How about your product owners? Do your network team know why they're seeing a sudden uplift in XSS probes on the web application firewall? Have you made your compliance teams aware? You're going to want to engage various teams early as possible, and feed back with the wonderful metrics you're gathering (see Tip 3) regularly.
+
+## Tip 9 - Partner Up
+Stick to what you're good at and don't try and do everything yourself. Dedicated bug bounty management platforms such as HackerOne (www.hackerone.com) can prove invaluable. They're not free but they offer solid tooling as well as guidance on programme operation and services such as mediation of any challenges or disagreements with hackers.
```

Comment:
Thanks Colin. The HackerOne link format is actually deliberate and I prefer the former. Don't think we have any style guidelines on this so happy to leave that one as-is

Point taken on images, I'm always loathe to try adding images due to attribution and copyright but I'll see what I can do